pkg/ebpf/event_parameters.go

```diff
@@ -0,0 +1,97 @@
+package ebpf
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/aquasecurity/tracee/pkg/ebpf/probes"
+	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/filters"
+	"github.com/aquasecurity/tracee/pkg/logger"
+)
+
+type eventParameterHandler func(t *Tracee, eventParams []map[string]filters.Filter[*filters.StringFilter]) error
+
+var eventParameterHandlers = map[events.ID]eventParameterHandler{
+	events.SuspiciousSyscallSource: attachSuspiciousSyscallSourceProbes,
+}
+
+// handleEventParameters performs initialization actions according to event parameters,
+// specified using policies or the command line as event arguments.
+// For example, an event can use one of its parameters to populate eBPF maps,
+// or perhaps attach eBPF programs according to the parameters.
+func (t *Tracee) handleEventParameters() error {
+	// Iterate through registerd event parameter handlers
+	for eventID, handler := range eventParameterHandlers {
+		// Make sure this event is selected
+		if _, err := t.eventsDependencies.GetEvent(eventID); err != nil {
+			continue
+		}
+		// Get the event parameters for all policies
```

Comment:
Let's add a comment about using data filters to get event prameters. We should add a TODO here that event parameters should be extracted from a dedicated parameteres field in the future